### ISSUE TITLE
fparser2 loses include statements

### ISSUE THREAD

**Issue by arporter** [stfc/fparser#138]
Given the following example:

    program test_include

      include 'this_file_cannot_be_found.h'

      write(*,*) "hello"
  
    end program test_include

fparser2 produces:

    $ scripts/fparser2.py ~/fort_include.f90 
    PROGRAM test_include
      WRITE(*, FMT = *) "hello"
    END PROGRAM test_include

**Comment by arporter** [comment]
Include files are handled by `FortranReaderBase.next()`:

                if not os.path.isfile(path):  # include file does not exist
                    dirs = os.pathsep.join(include_dirs)
                    # According to Fortran standard, INCLUDE line is
                    # not a Fortran statement.
                    message = '{!r} not found in {!r}. ' \
                              + 'INLCUDE line treated as comment line.'
                    reader.warning(message.format(filename, dirs), item)
                    item = self.next(ignore_comments)
                    return item

**Comment by arporter** [comment]
The comment in the above code is correct, section 3.4 of the F2003 standard explicitly states that Include is not a Fortran statement.

**Comment by arporter** [comment]
However, in order not to lose INCLUDE statements (for which the associated file cannot be found), we would need to represent them in the AST.

**Comment by rupertford** [comment]
Created branch - lose_includes

**Comment by rupertford** [comment]
The reader now outputs any include statements that can't be found rather than throwing them away.
I've also fixed the logging errors that were happening with some tests on some machines. This was fixed by setting the logging level explicitly in the code.

Now I *just* have the problem of the parser not understanding what an include statement is. This would be relatively simple but I am not sure how best to allow includes to exist at arbitrary locations. I could add it to all relevant rules but there are many of them. Perhaps I should add it in a similar way to the way Andy has done for comments? Perhaps I can just add it to an appropriate base class?
